.travis.yml

```diff
@@ -3,10 +3,13 @@ before_install:
   - sudo -E apt-get -yq --no-install-suggests --no-install-recommends install nginx-light snapd python3-venv
```

Comment:
Since the `snap` plugin in Travis is enabled, `snapd` is already installed. However `apt-get` has still an effect here, since in upgrade `snapd` to the latest version available. So I think it does not harm to keep `snapd` as a dependency to install.

Reply:
Thank you for pointing this out. I didn't consider this. I prefer to remove things that aren't strictly necessary to keep the whole thing simple, so I propose to remove snapd from the list of packages to install. CI will tell us if it works, and the lxd images are kept fairly up to date anyway. Any objection?

Reply:
No objection, if it works with the snapd version installed by Travis.